samples/scenario/list-namespaces.yaml

```diff
@@ -15,4 +15,4 @@
       concurrency: 4
     sla:
       failure_rate:
-        max: 0
\ No newline at end of file
+        max: 0
```

Comment:
Also, it would be nice to use task format v2. It allows to leave titles and descriptions. See https://github.com/xrally/xrally-docker/blob/master/samples/tasks/scenarios/docker-list-networks.yaml